mmv1/third_party/tgc_next/test/assert_test_files.go

```diff
@@ -169,7 +170,19 @@ func testSingleResource(t *testing.T, testName string, testData ResourceTestData
 			if err := compareAssetName(asset.Name, roundtripAsset.Name); err != nil {
 				return err
 			}
-			if diff := cmp.Diff(asset.Resource, roundtripAsset.Resource); diff != "" {
+			if diff := cmp.Diff(
+				asset.Resource,
+				roundtripAsset.Resource,
+				cmpopts.IgnoreFields(caiasset.AssetResource{}, "Version", "Data"),
+				// Consider DiscoveryDocumentURI equal if they have the same number of path segments when split by "/".
```

Comment:
Do we want to compare DiscoveryDocumentURI at all? It's not really a property of the resource itself.